declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="sharding-clusters-across-argo-cd-application-controller-replicas"]
+= Sharding clusters across Argo CD Application Controller replicas
+:context: sharding-clusters-across-argo-cd-application-controller-replicas
+
+toc::[]
+
+You can shard clusters across multiple Argo CD Application Controller replicas if the controller is managing too many clusters and uses too much memory.
+
+include::modules/gitops-argo-cd-round-robin-sharding-algorithm.adoc[leveloffset=+1]
+
+//include the dynamic scaling here after the round-robin procedure
```

Comment:
I think this structure makes the most sense as we will be also adding  the commented dynamic scaling section. I created the module based on the inspiration here: [Installing OLM Operators using Red Hat OpenShift GitOps
](https://docs.openshift.com/gitops/1.10/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#gitops-installing-olm-operators-using-gitops_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations) and its subsections: [Installing cluster-scoped Operators](https://docs.openshift.com/gitops/1.10/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#installing-cluster-scoped-operators_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations) and [Installing namespace-scoped Operators](https://docs.openshift.com/gitops/1.10/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#installing-namespace-scoped-operators_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)